browser/ui/toolbar/brave_location_bar_model_delegate.cc

```diff
@@ -32,6 +35,17 @@ void BraveLocationBarModelDelegate::FormattedStringFromURL(const GURL& url,
         base::UTF8ToUTF16("brave://"));
   }
 
+#if BUILDFLAG(IPFS_ENABLED)
```

Comment:
@bbondy Im not sure if we need it here, the location bar keeps brave://ipfs instead of brave://ipfs-internals  without it, is this ok to have brave://ipfs in the location bar?

Reply:
I think we should have this but just replace it with ipfs-internals instead of ipfs.  
Otherwise I think it retains chrome://ipfs doesn't it? Maybe when you edit the URL bar? I don't remember exactly.

Reply:
`const char kIPFSHost[] = "ipfs-internals";`